### PR TITLE
Doc-2696-Rearranged the CCM folders

### PR DESCRIPTION
**Pull request by charanya-j**
Restructured the left nav for the CCM module in the HDH.

Preview build: 
https://6433b5acc1587d571fd1dd4b--harness-developer.netlify.app/docs/cloud-cost-management

### CODE REVIEW

File: docs/cloud-cost-management/8-monitor-cloud-cost-with-integration/_category_.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"label": "Integration with third-party tools", "collapsible": "true", "collapsed": "true", "className": "red", "link": {"type": "generated-index", "title": "Integration with third-party tools"}, "customProps": {"position": "30", "helpdocs_category_id": "8a7gmuvz7e"}}
```
Comment (deepakbolangady) [inline docs/cloud-cost-management/8-monitor-cloud-cost-with-integration/_category_.json:1]: Can we shorten to "Third-party integrations"?

File: docs/cloud-cost-management/8-monitor-cloud-cost-with-integration/datadog-integration.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Datadog Integration with Cloud Cost Management
+title: Monitor cloud costs with Datadog integration
```
Comment (deepakbolangady) [inline docs/cloud-cost-management/8-monitor-cloud-cost-with-integration/datadog-integration.md:2]: Given that the parent is titled "Integration with third-party tools" (or "Third-party integrations" if you incorporate my other comment), it should be sufficient to shorten this title to "Datadog".

File: docs/cloud-cost-management/2-getting-started-ccm/3-use-cloud-integration.md [hunk 3]
```diff
@@ -12,8 +12,8 @@ Cloud Integration allows you to seamlessly set up your Harness CCM with a single
 
 ## Prerequisites and Permissions
 
-* See [Roles and Policies for the Connector](../../../platform/7_Connectors/add-a-kubernetes-cluster-connector.md#review-roles-and-policies-for-the-connector) to learn about the IAM roles and policies that you need to be assigned to be able to create a connector.
-* See [Prerequisites in Set up CCM for Kubernetes](../../1-onboard-with-cloud-cost-management/set-up-cloud-cost-management/set-up-cost-visibility-for-kubernetes.md) to learn about the tasks that you need to perform before setting up Cloud Costs Management for your Kubernetes cluster.
+* See [Roles and Policies for the Connector](../../../docs/platform/7_Connectors/add-a-kubernetes-cluster-connector.md#review-roles-and-policies-for-the-connector) to learn about the IAM roles and policies that you need to be assigned to be able to create a connector.
+* See [Prerequisites in Set up CCM for Kubernetes](../2-getting-started-ccm/4-set-up-cloud-cost-management/set-up-cost-visibility-for-kubernetes.md#prerequisites) to learn about the tasks that you need to perform before setting up Cloud Costs Management for your Kubernetes cluster.
```
Comment (deepakbolangady) [inline docs/cloud-cost-management/2-getting-started-ccm/3-use-cloud-integration.md:16]: Global:
* Does the preposition "in" need to be part of the link text?
* Referring to "Set up CCM for Kubernetes": the parent is "Set Up Cloud Cost Management", so can we reduce the child topics to one word each? For example, "AWS" and "Kubernetes"? Even the parent category can use the acronym.

File: docs/cloud-cost-management/2-getting-started-ccm/quick-start-guides/_category_.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"label": "CCM quickstart guides", "collapsible": "true", "collapsed": "true", "className": "red", "link": {"type": "generated-index", "title": "CCM quickstart guides"}, "customProps": {"position": "20", "helpdocs_category_id": "iul2qmg1yk"}}
```
Comment (deepakbolangady) [inline docs/cloud-cost-management/2-getting-started-ccm/quick-start-guides/_category_.json:1]: Shorten to "Quickstarts"?

**Review by deepakbolangady — APPROVED** [review]
My feedback is currently more on the length of each title. We should be able to reduce the amount of text. Please take a look at the feedback. Long titles also lead to long breadcrumbs at the top of each page (for example, see the breadcrumb at the top of the article "Enable Cloud Cost Management on your Kubernetes clusters". 

One comment I have that's out of the PR's scope is about why "CCM Auditing" is in the "get started" category. Do we need to reconsider its location? Please check (not necessarily as part of this PR).

Approving with the understanding that you will consider and incorporate this feedback. Let me know if I need to approve again.

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md [hunk 5]
```diff
@@ -32,7 +29,7 @@ To use Cost Categories, your Harness User account must belong to a User Group wi
 
 * **Cloud Cost Management**: **Cost Categories**: **Create/Edit**
 
-For more details, go to [CCM Roles and Permissions](../../4-cloud-cost-technical-reference/ccm-ref/ccm-roles-and-permissions.md).
+For more details, go to [CCM Roles and Permissions](../../2-getting-started-ccm/5-access-control/ccm-roles-and-permissions.md).
```
Comment (deepakbolangady) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md:32]: I missed capitalization issues in my earlier review. Can we fix them across the board? Use sentence case.

**Review by deepakbolangady — CHANGES_REQUESTED** [review]
I missed the capitalization inconsistencies in the left nav (thanks to Kim for pointing it out). Please fix them and then proceed). 
Also, reconsider the CCM auditing topic's location. Not wanting too many top-level categories in a doc is not reason enough to place it in the get started section.

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md (hunk 5 above)
Comment (charanya-j) [reply docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md:32]: I have fixed the headings. Thank you.

**Pull request merged**